base_layer/core/src/chain_storage/blockchain_database.rs

```diff
@@ -1726,6 +1805,24 @@ fn restore_reorged_chain<T: BlockchainBackend>(
         insert_block(&mut txn, block)?;
     }
     db.write(txn)?;
+
+    // Update reorg info
+    if last_reorg_best_block.is_some() {
```

Comment:
This is already done in `rewind_to_height`, is this duplicate code?

Reply:
Not duplicate code, different pathways/edge cases catered for here.